app/views/organizations/staff/dashboard/index.html.erb

```diff
@@ -1,75 +1,103 @@
 <%= render DashboardPageComponent.new do |c| %>
   <% c.with_header_title { t("dashboard.index.header_title") } %>
   <% c.with_body do %>
-    <div class="row">
+    <div class="row g-4">
       <div class="col-xl-3 col-lg-6 col-md-12 col-12">
-        <%= render DataCardComponent.new do |card| %>
-          <% card.with_title { "Tasks" } %>
-          <% card.with_icon_name { "list" } %>
-          <% card.with_value do %>
-            <div>
+        <div class="card h-100 shadow-sm border-0"> 
+          <div class="card-body d-flex flex-column justify-content-between">
+            <div class="d-flex align-items-center mb-3">
+              <span class="icon-shape bg-primary text-white rounded-circle p-2">
+                <i class="fe fe-list"></i>
+              </span>
+              <h4 class="ms-3 mb-0 text-primary">Tasks</h4> 
+            </div>
+            <div class="fs-4 ms-1">
               <div>
-                <%= link_to "#{@not_completed_not_overdue_tasks_count} Incomplete",
-                pets_with_incomplete_tasks_staff_dashboard_index_path,
-                  data: { turbo_frame: "tasks-frame" }
-                %>
+                <%= link_to pets_with_incomplete_tasks_staff_dashboard_index_path, data: { turbo_frame: "tasks-frame" }, class: "text-decoration-none text-gray-500 fw-semibold" do %> 
+                  <i class="fe fe-alert-circle text-primary me-1"></i>
+                  <%= "#{@not_completed_not_overdue_tasks_count} Incomplete" %>
+                <% end %>
               </div>
               <div>
-                <%= link_to "#{@not_completed_overdue_tasks_count} Overdue",
-                pets_with_overdue_tasks_staff_dashboard_index_path,
-                  data: { turbo_frame: "tasks-frame" }
-                %>
+                <%= link_to pets_with_overdue_tasks_staff_dashboard_index_path, data: { turbo_frame: "tasks-frame" }, class: "text-decoration-none text-gray-500 fw-semibold" do %>
+                  <i class="fe fe-clock text-primary me-1"></i>
+                  <%= "#{@not_completed_overdue_tasks_count} Overdue" %>
+                <% end %>
               </div>
             </div>
-          <% end %>
-        <% end %>
+          </div>
+        </div>
       </div>
       <div class="col-xl-3 col-lg-6 col-md-12 col-12">
-        <%= render DataCardComponent.new do |card| %>
-          <% card.with_title { "Applications" } %>
-          <% card.with_icon_name { "file-text" } %>
-          <% card.with_value do %>
-            <div>
+        <div class="card h-100 shadow-sm border-0">
```

Comment:
Ah - this highlights a deficiency with our DataCardComponent. Ideally we should be able to pass it a `title` and a `body` so that we can use it with these new cards you have made. 

We use the DataCardBody component only in the staff and adopter foster dashboards, both of which will change, so there's no existing logic to handle.

It's your call - do you want to address that in this PR, or would you rather I make a new issue for it?

Reply:
Thank you @kasugaijin.

I updated code to use DataCardComponent for title and icon but could not customize color for title and icon as per earlier design.

Here how it looks now

![staff-dashboard-stats](https://github.com/user-attachments/assets/06b3f685-ce07-4d2e-9d9a-0c5cd2694789)

Reply:
Do you want to go ahead with this or should I update code for DataCardComponent to customize css for title and icon as well?

Reply:
I think it would be good to have some pre-set options for the title size e.g., classes for `:small` `:medium` `:large` and some colour options for the icon. I think we should keep the title colour as it is for now, so it's consistent across the app. What do you think?

Reply:
I implemented title size and icon color attribute but kept it to minimum. I did not added other classes like "icon-shape rounded-circle p-2" as it was not looking good with current presentation of icon on right side.

This is how it looks with current updates

![staff-dashboard-stats](https://github.com/user-attachments/assets/e39f652d-8440-43a1-a08b-720fe0298578)

Reply:
That look great I think!